.gitignore

```diff
@@ -12,3 +12,4 @@ web-build/
 
 # macOS
 .DS_Store
+.expo
```

Comment:
@nbonatsakis is this okay?

Reply:
Yeah I don't think we want that checked in.

Reply:
Yes, it's okay to keep .expo out of the git history.

Here is the `.gitignore` file from the official expo repo. https://github.com/expo/expo/blob/a4fe2721a719adb95f5e296d29a77e2befc11819/.gitignore#L2